### DISCUSSION TITLE
[NEWS] AzerothCore ❤️ Docker

### DISCUSSION THREAD

**Discussion by Yehonal** [azerothcore/azerothcore-wotlk#5627] [News & Blog]
After the [first PR](https://github.com/azerothcore/azerothcore-wotlk/pull/4488) about the refactoring of the docker system, I received a lot of feedback from the AC community so I worked hard these days to fix all of them in the following PRs 

https://github.com/azerothcore/azerothcore-wotlk/pull/5551
https://github.com/azerothcore/wiki/pull/444
https://github.com/azerothcore/azerothcore-wotlk/pull/5444
https://github.com/azerothcore/azerothcore-wotlk/pull/5432

A special thank to Locus, Einhärjar, ReynoldsCahoon, and Eldarix

**The idea behind this work is to make AC 100% accessible to EVERYONE without any programming skills with [a small getting started](https://github.com/azerothcore/acore-docker/blob/master/README.md#getting-started)  (we hate repacks), and from there you can start your [journey with AC](https://www.azerothcore.org/wiki/Install-with-Docker) when you will be ready!**

There are tons of new things you can do with the new docker layer, just to mention one of the coolest:

You DO NOT have any more to configure several different workspaces and work differently or limited by your OS. AzerothCore now integrates full support for the VSCode dev-container feature and, together with the ccache integration, that's what you can do now and it [runs everywhere!](https://en.wikipedia.org/wiki/Write_once,_run_anywhere)

Video: https://www.youtube.com/watch?v=3NTNApJiyvo

[![IAC with Dev Container](https://img.youtube.com/vi/3NTNApJiyvo/0.jpg)](https://www.youtube.com/watch?v=3NTNApJiyvo)

More videos are coming, especially for our new docker repo

This is the list of changes:

- refactoring documentation for the docker installation 
- Integrated [Deno](https://deno.land/) in our dashboard. ❤️ TS
- differentiate local vs production images and push them both on docker hub
- build cache between authserver and worldserver
- fix errors in build stage that also fixes the cache hit problem
- integrated ccache inside the dashboard compiler
- docker CI now caches the ccache
- Improved VSCode dev-container integration (added tasks)
-  **🌒 Integrated Eluna within our production images allowing everyone to customize AC even with pre-compiled containers.**
- Solved the issues related to this project: https://github.com/azerothcore/azerothcore-wotlk/projects/28 , please assign any docker issue there
- adapting new changes to the acore-docker repo: https://github.com/azerothcore/acore-docker

Stay tuned!

**Comment by FrancescoBorzi** [comment]
excellent job! [who needs repacks nowadays?](https://www.azerothcore.org/acore-docker)
